packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -226,6 +237,67 @@ export const PaymentForm = ({
       </>
     );
 
+  const buttons = onCancel ? (
+    <div className="button-row">
+      <Localized id="payment-cancel-btn">
+        <button
+          data-testid="cancel"
+          className="settings-button cancel secondary-button"
+          onClick={onCancel}
+        >
+          Cancel
+        </button>
+      </Localized>
+
+      <Localized
+        id={submitButtonL10nId ? submitButtonL10nId : 'payment-update-btn'}
+      >
+        <SubmitButton
+          data-testid="submit"
+          className="settings-button primary-button"
+          name="submit"
+          disabled={inProgress}
+        >
+          {inProgress ? (
+            <span data-testid="spinner-update" className="spinner">
+              &nbsp;
+            </span>
+          ) : submitButtonCopy ? (
+            <span>{submitButtonCopy}</span>
+          ) : (
+            <span>Update</span>
+          )}
+        </SubmitButton>
+      </Localized>
+    </div>
+  ) : submit ? (
+    <div className="button-row">
+      <Localized id="payment-submit-btn">
+        <SubmitButton
+          data-testid="submit"
+          name="submit"
+          disabled={!allowSubmit}
+        >
+          {showProgressSpinner ? (
+            <span data-testid="spinner-submit" className="spinner">
+              &nbsp;
+            </span>
+          ) : (
+            <Localized
+              id={
+                submitButtonL10nId
+                  ? submitButtonL10nId
+                  : payButtonL10nId(customer!)
+              }
+            >
+              <span className="lock">{submitButtonCopy}</span>
+            </Localized>
+          )}
+        </SubmitButton>
+      </Localized>
+    </div>
+  ) : null;
```

Comment:
When do we want `buttons` to be `null`?

Reply:
This case is hit for when `submit` is false. In that case we would not want to show the submit button and buttons should be null.

This was left over from when I was going to provide my own submit button at the parent level. It's no longer valid or needed, I will remove.